src/Bicep.LangServer/LangServerResources.resx

```diff
@@ -117,7 +117,25 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="DeploymentFailedMessage" xml:space="preserve">
+    <value>Deployment failed.</value>
+  </data>
+  <data name="DeploymentFailedWithExceptionMessage" xml:space="preserve">
+    <value>Deployment failed. {0}</value>
+  </data>
+  <data name="DeploymentSuccessfulMessage" xml:space="preserve">
+    <value>Deployment successful.</value>
```

Comment:
Did you mean `Deployment succeeded`?

Reply:
Not a big deal, but it does make it more consistent for failure and success to both be verbs.

Reply:
Updated as part of this commit:
https://github.com/Azure/bicep/commit/32bed56af070ae3b2c2b494c956f07cf68874667

Thanks!